test/test_quantized.py

```diff
@@ -273,6 +273,50 @@ def test_cat(self, X, num, axis, relu):
             cat_q = q_cat_op(tensors_q, axis=axis, scale=scale,
                              zero_point=zero_point)
 
+
+@unittest.skipIf(
+    TEST_WITH_UBSAN or not torch.fbgemm_is_cpu_supported(),
+    " Quantized Linear requires FBGEMM. FBGEMM does not play"
+    " well with UBSAN at the moment, so we skip the test if"
+    " we are in a UBSAN environment.",
+)
+class TestDynamicQuantizedLinear(TestCase):
+    """Tests the correctness of the dynamic quantized linear and linear_relu op."""
+    @given(
+        use_bias=st.booleans(),
+        use_relu=st.booleans(),
+    )
+    def test_qlinear(self, use_bias, use_relu):
+        batch_size = 1
```

Comment:
Please randomize over batch size, input and output channels. We should check that:
 ```python
# Observe X_fp32 and determine scale and zero-point, this should match internals of dynamic linear.
 X_q = torch.quantize_linear(X_fp32 , scale , zero_point)
Y_fp32_ref = F.linear(X_q.dequantize(), W_q.dequantize(), b_fp32)
Y_fp32 = qlinear_dynamic(X_q.dequantize(), W_prepack,b_fp32)
```

Reply:
Thanks @raghuramank100 . As discussed in https://github.com/pytorch/pytorch/pull/23104, we created a task for the unit test for the exact match between static and dynamic quantization. Currently, if we use a randomized tensor as you suggested, we cannot get an exact match with `assert_equal`.